api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -433,26 +434,27 @@ private Workspace updateWorkspaceImpl(
     workspaceService.enforceWorkspaceAccessLevelAndRegisteredAuthDomain(
         fromWorkspaceNamespace, fromWorkspaceId, WorkspaceAccessLevel.READER);
 
-    DbUser user = userProvider.get();
```

Comment:
moved this lower because we need the DbAccessTier in the "from" DbWorkspace.

TODO (RW-6180) - enforce matching Tiers for From and To workspaces